org.mozilla.Firefox.appdata.xml

```diff
@@ -49,9 +46,9 @@
   <description xml:lang="zh_TW"><p>Firefox 是世界上最熱門的網頁瀏覽器之一。若您在瀏覽需要 Java 支援的網站時遇到問題，就試試 Firefox 吧。** 需要網路</p></description>
 
   <screenshots>
-    <screenshot type="default">
-      <image>https://d3lapyynmdp1i9.cloudfront.net/screenshots/org.mozilla.Firefox/C/org.mozilla.firefox-screenshot1.jpg</image>
-    </screenshot>
+    <screenshot type="default">https://raw.githubusercontent.com/hughsie/fedora-appstream/master/screenshots-extra/firefox/a.png</screenshot>
+    <screenshot>https://raw.githubusercontent.com/hughsie/fedora-appstream/master/screenshots-extra/firefox/b.png</screenshot>
+    <screenshot>https://raw.githubusercontent.com/hughsie/fedora-appstream/master/screenshots-extra/firefox/c.png</screenshot>
```

Comment:
Wonder if we should keep the screenshot of it running on EndlessOS instead - and maybe update it to the new version.

Anyway, this is a detail we can check later, but please do it as a separate commit as mentioned before and we can then decide whether to include this change.